examples/vnet/doca_gw_dpdk.h

```diff
@@ -0,0 +1,187 @@
+#ifndef _DOCA_GW_DPDK_H_
+#define _DOCA_GW_DPDK_H_
+
+#include <stdio.h>
+#include <rte_flow.h>
+#include "doca_gw.h"
+#include "doca_utils.h"
+
```

Comment:
bill please add a prefix for each action/match/union or any data structure that is doca_dpdk